incubator/azuremonitor-containers/templates/omsagent-deployment.yaml

```diff
@@ -38,8 +38,15 @@ spec:
        resources:
 {{ toYaml .Values.omsagent.resources.deployment | indent 9 }}
        env:
+       {{- if ne .Values.omsagent.env.clusterId "<your_cluster_id>" }}
+       - name: AKS_RESOURCE_ID
+         value: {{ .Values.omsagent.env.clusterId | quote }}
+       - name: AKS_REGION
+         value: {{ .Values.omsagent.env.clusterRegion | quote }}
```

Comment:
same. better to check for region being present (like clusterid)

Reply:
> same. better to check for region being present (like clusterid)

Fixed.